package.json

```diff
@@ -230,6 +230,7 @@
       "order": 16
     },
     "opener": {
+      "Description": "Default opener to use when viewing PDF, PS or DVI files. `automatic` will try to find the viewer best suited to the file type and your current settings."
```

Comment:
This needs to be `"description"`, also this package.json is invalid since this line is also missing a comma at the end.